prometheus/alert_manager/alertmanager.yml

```diff
@@ -9,15 +9,24 @@ route:
   group_wait: 0s
   group_interval: 5m
   repeat_interval: 1h
-  
+ 
+
   routes:
-    - receiver: 'node-monitoring'
-      group_by: ['...']
+    - matchers:
+      - severity="major"
+      receiver: seinetwork-warning
 
-  receiver: node-monitoring
+    - matchers:
+      - severity="critical"
+      receiver: seinetwork-rules
+    
+  receiver: seinetwork-rules
 
 receivers:
-- name: 'node-monitoring'
-  webhook_configs:
-  - send_resolved: true
-    url: 'http://alertmanager-bot:8080'
\ No newline at end of file
+- name: 'seinetwork-rules'
+  pagerduty_configs:
+    - service_key: 'REPLACE-ME'
+
+- name: 'seinetwork-warning'
+  pagerduty_configs:
+    - service_key: 'REPLACE-ME'
```

Comment:
are we going to replace these with the actual values? Probably need to create new services in PagerDuty

Reply:
yea i created a new service and replace the service_key with the integ key on the instance, don't wanna leave the integ key there cuz people can just page us using it